run.sh

```diff
@@ -99,7 +99,7 @@ function tear_down() {
 }
 
 function run_tests() {
-   eval "./mvnw -Dtest=$TESTS  test"
+   eval "./mvnw -Dtest=$TESTS -DPLATFORM_TYPE=$PLATFORM test"
```

Comment:
@trisberg Any reason why we are passing via -D? Not a big deal, but all other properties are set via environment and not via command line, maybe it would be better to follow one approach only.

Also, I noticed we export PLATFORM but the tests expect PLATFORM_TYPE, shouldn't us just export PLATFORM_TYPE?